### PR TITLE
chore: change register prompts

### PR DESCRIPTION
**Pull request by hugorut**
Adds extra step in register for both regenerating and new users:

**First time registering - answers Y**
Screenshot 2022-02-07 at 15 19 16

**First time registering  - answers N**
Screenshot 2022-02-07 at 15 19 40

**Reregistering, says Y to API key then answers Y to docs**
Screenshot 2022-02-07 at 15 18 51

**Reregistering, says N to API key then answers Y to docs**
Screenshot 2022-02-04 at 16 11 43

### CODE REVIEW

**Comment by hugorut** [comment]
@alikhajeh1 can you make sure the text/flow is what you had in mind?

**Comment by alikhajeh1** [comment]
Almost there, when I ran this:
```
✗ Would you like to generate a new API key: N
✗ Would you like to see our CI/CD integration docs?: N
```

I expected to see the following, but the CLI just exited. Other than that it was solid. Thanks!
`You can now run infracost breakdown --path=... and point to your Terraform directory or JSON plan file.`

**Comment by hugorut** [comment]
@alikhajeh1, I'm not 100% sold on that messaging as it reads a bit weird to me. I.e. in the scenario which the users answers **N** to generating a new api key and **N** to view docs, they've most likely just run the command by accident, so showing them the breakdown help text feels a bit off. However, it's not something I feel super strongly about and It probably makes sense with the rest of the changes. Command prompts now looks like such:

Screenshot 2022-02-08 at 09 55 34

**Comment by alikhajeh1** [comment]
LGTM! Thanks @hugorut - I went with it as there's at least a clear action item for users but yeah it does feel a bit off. We'll be iterating on this area in the future for sure

File: cmd/infracost/register.go [hunk 1]
```diff
@@ -100,14 +125,18 @@ func registerCmd(ctx *config.RunContext) *cobra.Command {
 				return err
 			}
 
-			msg = fmt.Sprintf("%s\nYou can now run %s and point to your Terraform directory or JSON/plan file.",
-				fmt.Sprintf("Your API key has been saved to %s", config.CredentialsFilePath()),
-				ui.PrimaryString("infracost breakdown --path=..."),
-			)
+			fmt.Printf("This was saved to %s\n\n", config.CredentialsFilePath())
+			if ciInterest {
+				fmt.Printf("You can now add cost estimates to your pull requests: %s\n", ui.LinkStringf("https://infracost.io/cicd"))
```
Comment (aliscott) [inline cmd/infracost/register.go:130]:
```suggestion
				fmt.Printf("You can now add cost estimates to your pull requests: %s\n", ui.LinkString("https://infracost.io/cicd"))
```

**Review by aliscott — APPROVED** [review]
LGTM

**Pull request merged**